src/components/common/TagForm/TagForm.tsx

```diff
@@ -75,7 +75,7 @@ export default function TagForm({
           }}
         />
       </form>
-      <AppliedTags applyedTags={applyedTags} onRemove={onRemove} />
+      {applyedTags.length ? <AppliedTags applyedTags={applyedTags} onRemove={onRemove} /> : <></>}
```

Comment:
```suggestion
      {applyedTags.length 0 > && <AppliedTags applyedTags={applyedTags} onRemove={onRemove} />}
```

요렇게 나타낼 수도 있을 거가타요!! 단순 취향입니다 !!!